src/Adapter/Product/Combination/Update/CombinationRemover.php

```diff
@@ -94,6 +98,29 @@ public function removeCombination(CombinationId $combinationId): void
     }
 
     /**
+     * @param CombinationId[] $combinationIds
+     */
+    public function bulkDeleteCombinations(array $combinationIds): void
+    {
+        $failedIds = [];
+        foreach ($combinationIds as $combinationId) {
+            try {
+                $this->removeCombination($combinationId);
```

Comment:
Im abit worried about this one (performance)

Reply:
the only solution I see is to make the command less ambigous (meaning that command would be actually for deleting just this product combinations, while now you can in theory delete any combinations for any different products with one command). That way we could use bulkDelete method and afterwards refresh the default combination of product and we dont need to fetch each combination to identify the product, cuz product will always stay the same for that command)

Reply:
I think the solution you suggest is good 👍 

> command would be actually for deleting just this product combinations
I think it makes sense 👍 . We make a Command for our usecase. If someone wants to delete combinations that belong to different products, he can make another Command.

We know performance is an important part of combination value and usecase.